src/main/java/dev/tricht/lunaris/item/ItemParser.java

```diff
@@ -32,6 +33,47 @@ public Item parse() {
             itemType = statsPart.getWeaponType();
         }
 
+        //If the item is a weapon, parse damage and attackspeed stats for DPS calculation
+        if (itemType instanceof WeaponItem){
+            log.debug("Weapon found!");
+
+            Pattern digitPattern = Pattern.compile("(\\d+\\.?\\d*)");
+
+            for(String stat : parts.get(1)){
+                Matcher m = digitPattern.matcher(stat);
+                ArrayList<Double> digits = new ArrayList<>();
+                while(m.find()){
+                    digits.add(Double.parseDouble(m.group(1)));
+                }
+
+                if(stat.contains("Physical Damage")){
+                    ((WeaponItem) itemType).setPhysMin(digits.get(0));
+                    ((WeaponItem) itemType).setPhysMax(digits.get(1));
+                }
+                else if(stat.contains("Elemental Damage")){
+                    ((WeaponItem) itemType).setFireMin(digits.get(0));
+                    ((WeaponItem) itemType).setFireMax(digits.get(1));
+                    if(digits.size()>2){
```

Comment:
If I understand correctly, if an item has Fire and Lightning damage, this code will confuse the Lightning damage for Cold damage?

I think to fix this we could check the mods instead of the stats (unless there are items with innate elemental damage?)
Will be considerably more difficult to implement though

Reply:
If we don't want to differentiate between the different types of elemental damage and just display "Elemental DPS" then this is fine IMO.

If we want to display Phys+Elem+Fire+Lightning+Cold+Chaos+Total then more has to be done here. Yea I could get the Damage Elements from the Mod part, this wont really be more difficult it will a bit more work tho.

Do you want it to be more detailed?